nimbus-features/etpCoordinatorRefactor.yaml

```diff
@@ -15,7 +15,7 @@ features:
           enabled: false
       - channel: developer
         value: 
-          enabled: false
+          enabled: true
```

Comment:
So discussing with Orla, turns out we could turn this ON for all channel 🎉 So true by default, and true on beta!